docs-ref-services/latest/servicebus.md

```diff
@@ -18,399 +18,41 @@ Microsoft Azure Service Bus supports a set of cloud-based, message-oriented midd
 * [SDK source code](https://github.com/Azure/azure-sdk-for-python/tree/master/azure-servicebus)
 * [SDK reference documentation](https://docs.microsoft.com/python/api/overview/azure/servicebus/client?view=azure-python)
```

Comment:
These two links are broken, and from the keyvault, I guess we don't need them any more?

Reply:
yup  - removing them